model/scikit/dffml_model_scikit/scikit_model_scorer.py

```diff
@@ -0,0 +1,57 @@
+from dffml.base import config
+from dffml.util.entrypoint import entrypoint
+from dffml.source.source import SourcesContext
+from dffml.model import ModelContext, ModelNotTrained
+from dffml.accuracy import (
+    AccuracyScorer,
+    AccuracyContext,
+)
+
+
+class ScorerWillNotWork(Exception):
+    pass
+
+
+@config
+class SklearnModelAccuracyConfig:
+    pass
+
+
+class SklearnModelAccuracyContext(AccuracyContext):
+    """
+    Default scorer for Sklearn Models.
+    """
+
+    async def score(self, mctx: ModelContext, sctx: SourcesContext):
+        if not mctx._filepath.is_file():
```

Comment:
We should probably put this on the list of things that need to be exposed by models.

So far we already knew we would need to expose the equivalent of the `_estimator_type`. We should also be exposing the model's trained or not trained status. Tracking here: #1143